src/cli/main.py

```diff
@@ -0,0 +1,79 @@
+"""
+Usage: instantshare.py <command> [<args>...]
+       instantshare.py [-h | --help | -v | --version]
+       instantshare.py help <command>
```

Comment:
> Change the order

I don't see the advantage of that. The topmost command is the most commonly used one, so that should be at the top. After all, you don't type `instantshare --help` just to find out what you need to type to get help.

> change the name of the program to "instantshare"

Would have already done that, just wasn't sure whether or not python on windows would handle that, as Windows tends to be picky about filenames. So thanks for clarifying!